Lib/glyphsLib/builder/builders.py

```diff
@@ -749,6 +749,25 @@ def _valid_designspace(self, designspace, ufo_module):
                         )
                     )
                     setattr(source, name, getattr(source.font.info, name))
+        for axis in copy.axes:
+            if not axis.map:
+                if any(
+                    x
+                    in [
+                        val
+                        for source in copy.sources
+                        for name, val in source.location.items()
+                        if name is axis.name
+                    ]
+                    for x in [axis.minimum, axis.maximum]
+                ):
+                    if axis.minimum == axis.maximum:
+                        axis.map = [(axis.minimum, axis.maximum)]
```

Comment:
Ok the two values are the same so you can use either interchangeably but still, seeing [(axis.minimum, axis.maximum)] is jarring. Please use the same value on both sides of the mapping (so that it's clear that it's an "identity" mapping)